e2e/test/compatibility.cy.spec.js

```diff
@@ -116,17 +116,53 @@ describe("Embedding SDK: shoppy compatibility", () => {
     })
   })
 
-  it("should open user created dashboards", () => {
+  it("should open user created collection", () => {
```

Comment:
```suggestion
  it("should contain the user-generated collection in saved explorations", () => {
```

since we're not actually opening the User-Generated collection